app/components/moderation-list-row/component.js

```diff
@@ -39,35 +39,56 @@ export default Component.extend({
     latestAction: computed('submission.actions.[]', function() {
         return latestAction(this.get('submission.actions'));
     }),
 
+    // first three contributors to the preprint
     firstContributors: computed('submission.node.contributors', function() {
         return this.get('submission.node.contributors').slice(0, 3);
     }),
 
+    // count of contributors in-addition to the first three
     additionalContributors: computed('submission.node.contributors', function() {
         return this.get('submission.node.contributors.content.meta.pagination.total') - 3;
     }),
 
+    // check if it has been more than a day since preprint last transition
     gtDay: computed('submission.dateLastTransitioned', function() {
         return moment().diff(this.get('submission.dateLastTransitioned'), 'days') > 1;
     }),
 
-    relevantDate: computed('submission.dateLastTransitioned', 'gtDay', function() {
+    // date of preprint acceptance or rejection
+    acceptedRejectedtDate: computed('submission.dateLastTransitioned', 'gtDay', function() {
         return this.get('gtDay') ?
             moment(this.get('submission.dateLastTransitioned')).format('MMMM DD, YYYY') :
             moment(Math.min(Date.parse(this.get('submission.dateLastTransitioned')), Date.now())).fromNow();
     }),
 
-    // translations
-    statusTimeDate: computed('submission.reviewsState', 'gtDay', 'noActions', function() {
+    // date of preprint submission
+    submitDate: computed('submission.dateCreated', 'gtDay', function() {
+        return this.get('gtDay') ?
+            moment(this.get('submission.dateCreated')).format('MMMM DD, YYYY') :
+            moment(Math.min(Date.parse(this.get('submission.dateCreated')), Date.now())).fromNow();
+    }),
+
+    // translations for accepted and rejected records
+    acceptedRejectedStatusTimeDate: computed('submission.reviewsState', 'gtDay', 'noActions', function() {
         const i18n = this.get('i18n');
         const dayValue = this.get('gtDay') ? 'gtDay' : 'ltDay';
         const timeWording = this.get('noActions') ? `${dayValue}_automatic` : dayValue;
-        const labels = ACTION_LABELS[this.get('submission.reviewsState')][timeWording];
-        return i18n.t(labels, { timeDate: this.get('relevantDate'), moderatorName: this.get('latestActionCreator') });
+        const status = this.get('submission.reviewsState');
+        const labels = ACTION_LABELS[status][timeWording];
+        return i18n.t(labels, { timeDate: this.get('acceptedRejectedtDate'), moderatorName: this.get('latestActionCreator') });
+    }),
+
+    // translations for pending records
+    pendingStatusTimeDate: computed('submission.reviewsState', 'gtDay', 'noActions', function() {
+        const i18n = this.get('i18n');
+        const dayValue = this.get('gtDay') ? 'gtDay' : 'ltDay';
+        const labels = ACTION_LABELS.pending[dayValue];
+        return i18n.t(labels, { timeDate: this.get('submitDate') });
```

Comment:
It looks like this is the only place `submitDate` is used, is there a reason it's a computed property instead of a local variable? Computed properties are useful, but can make it harder to see how information flows.